Dockerfile

```diff
@@ -15,10 +15,14 @@ COPY . .
 
 RUN CGO_ENABLED=0 go build -o webhook -ldflags '-w -extldflags "-static"' .
 
-FROM alpine:3.9
+FROM alpine:3.20.0
```

Comment:
For base images we define only major and minor versions to have an updated version on a new build